include/dgl/runtime/parallel_for.h

```diff
@@ -26,11 +27,13 @@ namespace {
 struct DefaultGrainSizeT {
   size_t grain_size;
 
-  DefaultGrainSizeT() {
-    auto var = std::getenv("DGL_PARALLEL_FOR_GRAIN_SIZE");
+  DefaultGrainSizeT() : DefaultGrainSizeT(1) {}
+
+  explicit DefaultGrainSizeT(size_t default_grain_size) {
+    auto var = dgl::kDGLParallelForGrainSize;
 
     if (!var) {
```

Comment:
nit: avoid if not else, flip it to if else.

Reply:
fixed